### ISSUE TITLE
Error opening profiler file

### ISSUE THREAD

**Issue by movedoa** [BalticAmadeus/ProPeek#146]
Just tried your extension but i cant seem to open any profiler file.
I'm on 12.2.

I always get this error:

```
2024-01-30 17:06:46.699 [error] TypeError: Cannot read properties of undefined (reading 'CumulativeTime')
    at getTotalSessionTime (c:\Users\doa\.vscode\extensions\balticamadeus.pro-peek-1.1.1\out\services\parser\presentationData.js:33:79)
    at transformData (c:\Users\doa\.vscode\extensions\balticamadeus.pro-peek-1.1.1\out\services\parser\presentationData.js:12:30)
    at ProfilerService.parse (c:\Users\doa\.vscode\extensions\balticamadeus.pro-peek-1.1.1\out\services\profilerService.js:15:70)
    at new ProfilerViewer (c:\Users\doa\.vscode\extensions\balticamadeus.pro-peek-1.1.1\out\webview\ProfilerViewer.js:55:42)
    at c:\Users\doa\.vscode\extensions\balticamadeus.pro-peek-1.1.1\out\extension.js:61:9
    at d.h (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:141:140094)
    at d.$executeContributedCommand (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:141:140954)
    at l.S (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:150:10829)
    at l.Q (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:150:10595)
    at l.M (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:150:9685)
    at l.L (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:150:8764)
    at f.value (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:150:7567)
    at n.y (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:80:1902)
    at n.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:80:2119)
    at o.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:103:14039)
    at f.value (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:171:7903)
    at n.y (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:80:1902)
    at n.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:80:2119)
    at o.fire (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:103:14039)
    at MessagePortMain.<anonymous> (c:\Program Files\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:171:6183)
    at MessagePortMain.emit (node:events:513:28)
    at Object.MessagePortMain._internalPort.emit (node:electron/js2c/utility_init:2:367) vsc-profiler.profiler {"value":"BalticAmadeus.pro-peek","_lower":"balticamadeus.pro-peek"}
```

**Comment by PauliusKu** [comment]
Hi @movedoa ,
could you provide us the the profiler file which caused this error. If you cannot, I just need to know:

1. the version of the file you have (first number in the first line of the file)
2. What is written in the first node of call tree (block after 6th dot). Looks something like this `0 0 1 0.000000 0.005111`

**Comment by movedoa** [comment]
[Profiler_2024-02-01T171346.zip](https://github.com/BalticAmadeus/ProPeek/files/14127926/Profiler_2024-02-01T171346.zip)

**Comment by PauliusKu** [comment]
Thanks. It seams we have an issue with v4 profiler files. We'll fix the bug and hopefully release it with v1.2 next week.
